src/QsCompiler/CompilationManager/AssemblyLoader.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.Quantum.QsCompiler.ReservedKeywords;
 using Microsoft.Quantum.QsCompiler.Serialization;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
+using Microsoft.Quantum.QsCompiler.Transformations;
```

Comment:
Why does this file now need to reference the Transformations project? Same question for CodeActions.cs, ProjectManager.cs, and TypeChecking.cs.

Reply:
It's for the Utils class defined in Transformations. I couldn't find a better place to put it that all the C# projects could access it from.